subsys/bluetooth/Kconfig

```diff
@@ -39,6 +39,16 @@ config BT_CUSTOM
 
 endchoice
 
+if !BT_RECV_IS_RX_THREAD
+
+config SYSTEM_WORKQUEUE_STACK_SIZE
+	default 3092 if BT_MESH_GATT_CLIENT
+	default 2200 if BT_SETTINGS
+	default 2048 if BT_MESH
+	default 2048 if BT_AUDIO
```

Comment:
I thought we didn't want to do this?

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/Kconfig#L44
Refer to this usage